msg/vehicle_attitude.msg

```diff
@@ -2,7 +2,7 @@
 
 uint64 timestamp		# time since system start (microseconds)
 
-float32[4] q			# Quaternion rotation from NED earth frame to XYZ body frame
+float32[4] q			# Quaternion rotation from XYZ body frame to NED earth frame.
```

Comment:
Good catch, we're consistently using hamilton quaternion convention since https://github.com/PX4/Matrix/pull/34 which has the 4 properties:
- the first element is real [w, x, y, z]
- right-handedness i * j = k
- passive operation, vectors stay we just change frame
- q rotates from local to global q * [0; v_local] * q* = v_global